### ISSUE TITLE
Add vNIC traffic

### ISSUE THREAD

**Issue by DebianRoxx** [sexibytes/sexigraf#188]
In the "VMware Cluster Network Usage" dashboard, we have **vmnic** traffic.
It will be nice to have **vNIC** traffic per host to show the difference with EST/WEST traffic.

**Comment by rschitz** [comment]
By vNIC traffic you mean the vm traffic?

**Comment by DebianRoxx** [comment]
Yes :)

**Comment by rschitz** [comment]
you already have the vmnic traffic and i suppose you have dedicated nics for VM on your ESX rights?

**Comment by DebianRoxx** [comment]
No, it's not that.
I'm talking about vNICs, virtual NICs of VMs.
It would be nice to have a dashboard showing the sum of network traffic per ESXi or per cluster, whether it is purely internal to the ESXi (2 VM on the same host) or external.

The information is already present at the dashboard level "VMware All Cluster VM Stats", so it would be the aggregation of these. (vmw.$vcenter.$datacentre.$cluster.vm.$vm.$vm.fatstats.netUsage)

Translated with www.DeepL.com/Translator

**Comment by rschitz** [comment]
good idea indeed, i'll try that and see how it goes on multiple clusters to see if that's relevant

**Comment by rschitz** [comment]
![image](https://user-images.githubusercontent.com/8385646/59277648-dfbd8780-8c60-11e9-83b0-fd67240d1d84.png)

**Comment by rschitz** [comment]
It's the up+down vNic Usage but it looks interesting anyway so we'll add it in the next release ;)

**Comment by DebianRoxx** [comment]
Nice ! :)
If you want a test subject, I'm here ;)

**Comment by rschitz** [comment]
During testing i figured out that it's impossible to compare the vmnic and the vnic traffic to deduce the internal only traffic because the vmnic traffic would include vsan, iscsi and nfs therefore misleading the comparaison.

Besides, having an in+out vm traffic metric in a graph where the host traffic is splitted between in and out bother me. I think it's not clear to read.

Mind to share youre thoughts about that?

**Comment by DebianRoxx** [comment]
Maybe we should put an information to warn the administrators ? But hey, that's not great...

**Comment by rschitz** [comment]
can't think of a right way to showing it so we'll just comment the code and think about it in the future

**Comment by rschitz** [comment]
https://github.com/sexibytes/sexigraf/commit/ffd6278867f83416fe8f4e9ab37eff3ca9351715
